www/md_base/src/app/builds/builder/forcedialog/forcedialog.controller.coffee

```diff
@@ -8,7 +8,7 @@ class ForceDialog extends Controller
 
     confirm: ->
         param = builderid: @builder.builderid
-        _.extend param, @fields
+        _.extend param, @data
```

Comment:
How many times do you use underscore? I think angular.extend/.merge is just fine here.